roles/aws/delete-instance/tasks/getinstance.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Get instance facts
+  ec2_instance_facts:
+    filters:
+      "tag:Name": "{{ instance.hostname_prefix }}.{{ vpc_name }}.{{ dns_domain }}"
+    region: "{{ aws_region }}"
+  register: instances
+  failed_when: "{{ instances|length == 0 }}"
+
+- debug:
```

Comment:
Looks like a left-over debug? At least set a level on it so it doesn't always print.

Reply:
Maybe just check on all the `debug` prints in the implementation

Reply:
Debug statements have been removed.